email-log.php

```diff
@@ -241,12 +351,9 @@ function display_content_callback() {
 	}
 
 	/**
-	 * Save Screen option.
+	 * Saves Screen options.
 	 *
-	 * @param unknown $status
-	 * @param unknown $option
-	 * @param unknown $value
-	 * @return unknown
+	 * @since Genesis
 	 */
 	function save_screen_options( $status, $option, $value ) {
```

Comment:
_Thanks for the comments_.

I was able to figure out what the `$option` and `$value` argument does but wasn't really sure about **`$status`**.

I tried to find from [web](https://pippinsplugins.com/add-screen-options-tab-to-your-wordpress-plugin/) yet couldn't figure it out. Could you please help?